src/About.js

```diff
@@ -5,7 +5,7 @@ import './About.css';
 class About extends Component{
     render(){
         return(
-            <div id="about" className="About">
+            <section id="about" className="About">
                 <div className="About-cover"/>
                 <fieldset className="About-fieldset">
                     <legend>
```

Comment:
todo: `fieldset` and `legend` is used for form elements. When we need to style something, we must always use CSS, as the default browser styles might even change from browser to browser. You can replicate a similar effect using only CSSHere's [a YT tutorial](https://www.youtube.com/watch?v=NkYoo2V9FKQ&ab_channel=NileshDadheech).

Reply:
Instead of `fieldset` and `legend` , I have used `h1` now and updated CSS based on this video. I have pushed the changes. Do let me know if ok.

Reply:
The line effect is cool, well done! 🥳 

Regarding the `h1` itself, have you read [this article about headings](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)? Some of the quotes there:

> - Use one unique h1 per page that describes what that page is about. That h1 preferably starts just above the main content.
> - Use heading levels like the index of a book: hierarchical.

There's this cool extension called [headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en), that lets you inspect the headings in the page. In your case you  [have this structure](https://user-images.githubusercontent.com/14869087/200118352-90855668-faf4-4696-a24e-2c50a11d0045.png).

My suggestion for you to fix that is:
- Change "About" and "Projects" to `<h2>`
- Move `h1` from "Hello" to "I'm Ghrismas"
  - Bonus: remember what I said in another comment about characters in uppercase? Here's another example for you to fix ☺️